libs/viewer/include/viewer/ViewerGui.h

```diff
@@ -256,7 +256,7 @@ class UTILS_PUBLIC ViewerGui {
     std::function<void()> mCustomUI;
 
     // Properties that can be changed from the UI.
-    int mCurrentAnimation = 1; // It is a 1-based index and 0 means not playing animation
+    int mCurrentAnimation = 0; // It is a 0-based index and -1 means not playing animation
```

Comment:
document that when set to the animation count it plays all of them (there are 2 special values, -1 and count).

Reply:
Updated.